components/reward-calculator/index.js

```diff
@@ -405,10 +407,10 @@ const RewardCalculatorPage = () => {
 			  stakingInfo
 			? setAmount(
 					Math.trunc(
-						(totalAvailableStakingAmount - networkInfo.reserveAmount) *
-							10 ** networkInfo.decimalPlaces
-					) /
-						10 ** networkInfo.decimalPlaces
+						((totalAvailableStakingAmount - networkInfo.reserveAmount) /
```

Comment:
@Lord-of-Codes  what is the logic behind this equation?

shouldn't it (`((totalAvailableStakingAmount - networkInfo.reserveAmount) / (1 + Number(networkInfo.feesRatio)))`) be as simple as: `((totalAvailableStakingAmount - networkInfo.reserveAmount) * (1 - networkInfo.feesRatio))`
Let me know if i am missing something here